snap/hooks/configure

```diff
@@ -21,10 +21,14 @@ if [ ! -f "${CONFIG_DIR}/initialized" ]; then
 
 	$SNAP/usr/sbin/mysqld --initialize --datadir=$CONFIG_DIR/data --user=snap_daemon
 
+        # Add default client config for exporter
+        sed -i '32s/^/\n[client]\nuser = exporter\npassword = exporterpasswd\nport = 3306\nhost = localhost\n\n/' $CONFIG_DIR/mysql.cnf
```

Comment:
This is not very impactful, but user will be changed to `monitoring` on the charm. So that could be reflected here as well.